### DISCUSSION TITLE
Cannot convert superset http to https

### DISCUSSION THREAD

**Discussion by shafad1** [apache/superset#22401] [Q&A / Help]
i have done the configuration in config.py 
SUPERSET_WEBSERVER_PROTOCOL = "https"
SUPERSET_WEBSERVER_ADDRESS = "0.0.0.0"
SUPERSET_WEBSERVER_PORT = 443

after done this configuration still superset is able to run with http only.. 
can anyone suggest the exact steps to solve this ,

**Comment by rusackas** [comment]
Converted this to a discussion since it does not represent a bug. Please also feel free to join the #deploying-superset channel on Slack if you'd like to take it up in that forum.
